src/js/utils/ErrorMessageUtil.js

```diff
@@ -0,0 +1,71 @@
+const MESSAGE_VARIABLE = /\{\{([^\}]+)\}\}/g;
+
+const ErrorMessageUtil = {
+
+  /**
+   * This function returns the `message` of the given array object,
+   * with the path and message components concatenated into a uniform
+   * string that can be displayed unanchored.
+   *
+   * @param {Object} error - The error message
+   * @param {Array} pathTranslationRules - The path translation rules
+   * @returns {String} Returns the composed error string
+   */
+  getUnanchoredErrorMessage(error, pathTranslationRules) {
+    const pathString = error.path.join('.');
+    const rule = pathTranslationRules.find(function (rule) {
+      return rule.match.exec(pathString);
+    });
+
+    // If no translation found, prefix with the path ID
```

Comment:
This comment looks like it describes line 25 so `return \`${pathString}: ${error.message}\``